server/controllers/userController.js

```diff
@@ -27,66 +28,88 @@ const signUp = asyncHandler(async (req, res) => {
     if (user) {
       const token = generateToken(user._id);
       res.cookie("dolphinToken", token, {
-        maxAge: 3600,
+        maxAge: 3600000,
+        sameSite: "none",
         httpOnly: true,
         secure: false, // should be true in Production !
       });
+
+      // Added for Co-op Midterm Presentation
+      // To be handled later on by BullMQ
+      const addMentionsToDB = async () => {
```

Comment:
I would suggest this be added to a utils/scraper file and imported that way.  Would be more clean and avoid having to copy/paste the logic throughout this file

Reply:
done !